nibabel/streamlines/__init__.py

```diff
@@ -0,0 +1,131 @@
+import os
+from ..externals.six import string_types
+
+from .header import Field
+from .compact_list import CompactList
+from .tractogram import Tractogram, LazyTractogram
+
+from .trk import TrkFile
+#from .tck import TckFile
+#from .vtk import VtkFile
+
+# List of all supported formats
+FORMATS = {".trk": TrkFile,
+           #".tck": TckFile,
+           #".vtk": VtkFile,
+           }
+
+
+def is_supported(fileobj):
+    ''' Checks if the file-like object if supported by NiBabel.
+
+    Parameters
+    ----------
+    fileobj : string or file-like object
+        If string, a filename; otherwise an open file-like object pointing
+        to a streamlines file (and ready to read from the beginning of the
+        header)
+
+    Returns
+    -------
+    is_supported : boolean
+    '''
+    return detect_format(fileobj) is not None
+
+
+def detect_format(fileobj):
+    ''' Returns the StreamlinesFile object guessed from the file-like object.
+
+    Parameters
+    ----------
+    fileobj : string or file-like object
+        If string, a filename; otherwise an open file-like object pointing
+        to a tractogram file (and ready to read from the beginning of the
+        header)
+
+    Returns
+    -------
+    tractogram_file : ``TractogramFile`` class
+        Returns an instance of a `TractogramFile` class containing data and
```

Comment:
Is there a list somewhere of all those markups (e.g. for functions, methods, parameters)?